yb-voyager/cmd/import.go

```diff
@@ -434,3 +444,18 @@ func validateTruncateTablesFlag() {
 		utils.ErrExit("Error --truncate-tables true can only be specified along with --start-clean true")
 	}
 }
+
+var onPrimaryKeyConflictActions = []string{
+	constants.PRIMARY_KEY_CONFLICT_ACTION_ERROR,
+	constants.PRIMARY_KEY_CONFLICT_ACTION_IGNORE,
+	constants.PRIMARY_KEY_CONFLICT_ACTION_UPDATE,
+}
+
+func validateOnPrimaryKeyConflictFlag() {
```

Comment:
I think we should prevent user from changing this during resumption. 
What if user starts out with IGNORE, there is a batch that succeeded partially. Now, if the user switches to ERROR, it will start failing?